src/modules/camCalibWithPose/src/CamCalibModule.cpp

```diff
@@ -166,12 +166,21 @@ bool CamCalibPort::selectBottleFromMap(double time,
     std::map<double, yarp::os::Bottle>::iterator it_prev, it_next;
 
     m.lock();
-    it_next = datamap->lower_bound(time);
+
+	it_next = datamap->lower_bound(time);
```

Comment:
Use spaces instead of tabs to avoid  this visualization issue on GitHub . See https://blogs.msdn.microsoft.com/zainnab/2010/09/08/insert-spaces-vs-keep-tabs/ .

Reply:
@hu-yue please respond to @traversaro's comment.